qa/L0_trace/test.sh

```diff
@@ -699,7 +707,7 @@ OTEL_COLLECTOR_LOG="./trace_collector_http_exporter.log"
 # Building the latest version of the OpenTelemetry collector.
 # Ref: https://opentelemetry.io/docs/collector/getting-started/#local
 if [ -d "$OTEL_COLLECTOR_DIR" ]; then rm -Rf $OTEL_COLLECTOR_DIR; fi
-git clone https://github.com/open-telemetry/opentelemetry-collector.git
+git clone --depth 1 --branch v0.82.0 https://github.com/open-telemetry/opentelemetry-collector.git
```

Comment:
@fpetrini15 This was the issue you were seeing. The current master branch has requirement for go to be at least 1.20 version.

Reply:
Added this edit to my setup and can confirm it resolved the error.